homeworks/01-bubble-sort/src/App.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {Component} from 'react'
+import Button from "./components/button/Button"
+import Column from "./components/column/Column"
+import './App.css'
+
+type AppState = {
+	statusOfGame: string,
+	statusOfBtn: string,
+	arrOfNumbers: number[],
+	arrOfTimerID: number[],
+	disabled: boolean,
+}
+
+class App extends Component<{}, AppState> {
+
+  private timerID: any
+
+	state = {
+		statusOfGame: 'not solved',
+		statusOfBtn: 'Start',
+		arrOfNumbers: [],
+		arrOfTimerID: [],
+		disabled: false,
+	}
+
+	componentDidMount() {
+		this.setState({arrOfNumbers: this.makeArr()})
```

Comment:
Guess, it will be one more redundant re-render. You can set it once during component initialization.

Reply:
fixed